src/Salas_de_video/Utils.java

```diff
@@ -14,4 +16,12 @@
         }
         return abecedario;
     }
+
+    public static ArrayList<String> listaPeliculas(Sala sala){
```

Comment:
que el nombre de la funcion sea mas claro

Reply:
El nombre esta claro, nos regresa la lista de peliculas de una sala (Claro solamente el nombre) Lo movi a la clase sala porque seria mas claro poner sala.listaPeliculas()

Reply:
```suggestion
    public static ArrayList<String> getListaPeliculas(Sala sala){
```

Esta funcion tendria mas sentido en el `RoomsManager` por que este es el encargado de manejar las salas.